coopr-server/pom.xml

```diff
@@ -82,6 +82,14 @@
             </exclusion>
           </exclusions>
         </dependency>
+
+      <!-- REMOVE -->
+      <dependency>
+        <groupId>mysql</groupId>
+        <artifactId>mysql-connector-java</artifactId>
+        <version>5.1.34</version>
+      </dependency>
```

Comment:
Not sure if we want to keep this, but it's helpful to make the `UpgradeTo0_9_9` script work against MySQL databases without having to bother with the classpath.